services/audit-service/src/main/java/datawave/microservice/audit/AuditController.java

```diff
@@ -122,17 +122,22 @@ private boolean sendMessage(AuditParameters parameters) {
             }
             
             boolean success = messageChannel.send(MessageBuilder.withPayload(AuditMessage.fromParams(parameters)).setCorrelationId(auditId).build());
-            
+
+            // Prevent a 2xx response code to the caller.
+            if (!success) {
+                throw new RuntimeException("Failed to send audit message " + auditId);
+            }
+
             if (auditProperties.isConfirmAckEnabled()) {
                 try {
-                    success = success && latch.await(auditProperties.getConfirmAckTimeoutMillis(), TimeUnit.MILLISECONDS);
+                    success = latch.await(auditProperties.getConfirmAckTimeoutMillis(), TimeUnit.MILLISECONDS);
```

Comment:
I agree that this change makes sense if we add the RuntimeException above, but per my previous comment, we should not start doing that.  So, I think this should remain as-is.  Otherwise, the message send failure will not carry forward.